README.md

```diff
@@ -54,8 +53,9 @@ without root privileges:
 It should be noted that `plugdev` can be changed to any Unix group of
 your preference.
 
-Designs
--------
+This should only be necessary when trying to access the bootloader.
```

Comment:
Is it? Surely the serial port will be owned by `root` by default? Depends on distro I guess if there's a dialout group configured.

Reply:
Linux handles serial ports separately to raw USB devices so if you have permissions to access serial ports in general then you will have access to this board.